apps/ui/src/App.css

```diff
@@ -109,3 +133,35 @@
   opacity: 1;
   animation: shimmer 1s running linear, close 1s ease-in-out 4s forwards;
 }
+
+
+
+.text-display {
+  text-align: center;
+    margin: 24px 0px;
+    background: linear-gradient(to right bottom, rgb(255, 255, 255) 30%, rgba(255, 255, 255, 0.38)) text;
+    -webkit-box-decoration-break: clone;
+    -webkit-text-fill-color: transparent;
+    color: unset;
+    padding-bottom: 0.13em;
+    font-size: 50px;
+    line-height: 1;
+    letter-spacing: -0.02em;
+    font-weight: 500;
+}
+
+.navigation-content {
+  display: flex;
+  flex-direction: row;
+  gap: 5px;
+  justify-content: space-between;
+}
+
+@media all and (max-width: 646px) {
+  .navigation-content {
+    justify-content: center;
+    display: flex;
+    flex-direction: column;
+    gap: 5px;
+  }
+}
```

Comment:
Add last line. I think the prettier plugin is not setup ideally based on current setup config for project, last line would be added automatically. Can we check this once in your local system?

Reply:
@ArnavNath550